library/core/src/task/wake.rs

```diff
@@ -866,7 +863,8 @@ impl LocalWaker {
     /// ```
     #[inline]
     #[must_use]
-    #[unstable(feature = "noop_waker", issue = "98286")]
+    #[stable(feature = "noop_waker", since = "CURRENT_RUSTC_VERSION")]
+    #[rustc_const_stable(feature = "noop_waker", since = "CURRENT_RUSTC_VERSION")]
```

Comment:
`LocalWaker` is itself still unstable, so I think this function should remain unstable, just under that feature gate?
```suggestion
    #[unstable(feature = "local_waker", issue = "118959")]
```

Reply:
Definitely, thanks for the suggestion!